test/unit/Models/account.spec.ts

```diff
@@ -0,0 +1,96 @@
+import {CGAccount} from "../../../src/renderer/models/account";
+import {ICGKeystore, ICGKeystoreFactory} from "../../../src/renderer/services/interfaces";
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+
+// Passwords for keystores 1 & 2
+const PRIMARY_KEYSTORE_PASSWORD = "chainGuardianPass";
+
+/*eslint-disable*/
+const CGKeystoreTest: ICGKeystoreFactory = class CGKeystoreTest implements ICGKeystore {
+    private file: string;
+    constructor(file: string) {
+        this.file = file;
+    }
+
+    decrypt(password: string): Keypair {
+        if (password === PRIMARY_KEYSTORE_PASSWORD) {
+            return Keypair.generate();
```

Comment:
this is faster:
https://github.com/NodeFactoryIo/ChainGuardian/pull/28/files#diff-f55bcd7f099136dc7b3589dd8b2728a1R6